### PR TITLE
Disco Floor

### PR DESCRIPTION
**Pull request by ps3moira**
## About the PR

Added disco floors.
## Why / Balance

Because it's fun to get your groove on 😎 
## Media
disco
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- add: Added disco floors. Groovy!

### CODE REVIEW

**Comment by leonardo-dabepis** [comment]
Another thing to stare at while high. I approve 👍

**Comment by IamVelcroboy** [comment]
No screenshot?

**Comment by ps3moira** [comment]
> No screenshot?

it's a draft and I'm in the middle of respriting the ones from SS13 to look better.

**Comment by ps3moira** [comment]
> No screenshot?

I added a gif of the updated version.

**Comment by IamVelcroboy** [comment]
Oh nice!

File: Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/structures/disco_tile.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      entity: DiscoFloor
+      edges:
+        - to: start
+          completed:
+            - !type:SpawnPrototype
+              prototype: SheetSteel5
+              amount: 5
+          steps:
+            - tool: Prying
+              doAfter: 5
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/structures/disco_tile.yml]: why have 2 of the same graph

Comment (ps3moira) [reply Resources/Prototypes/DeltaV/Recipes/Construction/Graphs/structures/disco_tile.yml]: Guh, I just realized how to get it to work without that. fixed!

**Pull request closed without merging**